addons/mail/static/src/model/tests/model_field_command_tests.js

```diff
@@ -0,0 +1,714 @@
+/** @odoo-module **/
+
+import {
+    clear,
+    create,
+    decrement,
+    increment,
+    insert,
+    insertAndReplace,
+    link,
+    replace,
+    unlink,
+    unlinkAll,
+} from 'mail/static/src/model/model_field_command.js';
+import {
+    afterEach,
+    beforeEach,
+    start,
+} from 'mail/static/src/utils/test_utils.js';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('model', {}, function () {
+QUnit.module('model_field_command_tests.js', {
+    beforeEach(){
+        beforeEach(this);
+        this.start = async params => {
+            const { env, widget } = await start(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
+    },
+    afterEach() {
+        afterEach(this);
+    },
+});
+
+QUnit.test('clear: should set attribute field undefined', async function (assert) {
```

Comment:
"...if there is no default value" (or in other words, this is actually just a particular case of default value, when the default value itself is undefined)